common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -1,42 +1,22 @@
 {
-    "env": {
-        "browser": true,
-        "es2024": false,
-        "es6": true,
-        "node": true
-    },
-    "globals": {
-        "Atomics": "readonly",
-        "SharedArrayBuffer": "readonly"
-    },
-    "ignorePatterns": [
-        "**/packageVersion.ts",
-        "**/layerGenerationState.ts",
-        "**/*.generated.ts",
-        "**/*.generated.js"
-    ],
-    "parserOptions": {
-        "ecmaFeatures": {
-            "jsx": true
-        },
-        "ecmaVersion": "latest",
-        "project": "./tsconfig.json",
-        "sourceType": "module"
+    "language": "@/js",
+    "linterOptions": {
+        "reportUnusedDisableDirectives": 1
     },
     "plugins": [
+        "@",
         "@eslint-community/eslint-comments",
-        "import-x",
+        "@typescript-eslint:@typescript-eslint/eslint-plugin@8.18.2",
         "depend",
-        "@fluid-internal/fluid",
-        "unused-imports",
-        "promise",
-        "jsdoc",
-        "@typescript-eslint",
+        "import-x:eslint-plugin-import-x@4.16.1",
+        "unicorn:eslint-plugin-unicorn@54.0.0",
         "@rushstack",
-        "unicorn",
-        "tsdoc"
+        "jsdoc",
+        "promise",
+        "tsdoc",
+        "unused-imports:unused-imports",
```

Comment:
Redundant?

Reply:
You mean the unused-imports plugin? Yeah possibly. At one point it did something that the import plugins didn't, but that is way outdated info.

Reply:
Ahh, I see what you mean. The string `unused-imports:unused-imports` does not appear anywhere in our source as far as I can tell. Only `unused-imports`